idd/Energy+.idd.in

```diff
@@ -34502,6 +34502,4630 @@ ZoneHVAC:EvaporativeCoolerUnit,
+	A20,	\field Mode0 Supply Air Humidity Ratio Lookup Table Name
+		\type object-list
+		\object-list MultiVariateTables
+		\note Enter the name of the Supply Air Humidity Ratio Lookup Table for Mode0.
+		\note If this field is blank, Mode0 will not be considered for any period that requires ventilation, heating, cooling, or dehumidification. If this field is blank, when Mode0 is chosen (during standby periods) the supply air humidty ratio will equal the return air humidity ratio.
+	A21,	\field Mode0 System Electric Power Lookup Table Name
+		\type object-list
+		\object-list MultiVariateTables
+		\note Enter the name of the Electric Power Lookup Table for Mode0.
+		\note If this field is blank, Mode0 does not consume electricity.
+	A22,	\field Mode0 Supply Fan Electric Power Lookup Table Name
+		\type object-list
+		\object-list MultiVariateTables
+		\note Enter the name of the Supply Fan Electric Power Lookup Table for Mode0.
+		\note If this field is blank, Mode0 does not consume electricity for supply fan.
+	A23,	\field Mode0 External Static Pressure Lookup Table Name
+		\type object-list
+		\object-list MultiVariateTables
+		\note Enter the name of the External Static Pressure Lookup Table for Mode0.
+		\note If this field is blank, external static pressure will not be reported.
+	A24,	\field Mode0 System Second Fuel Consumption Lookup Table Name
+		\type object-list
+		\object-list MultiVariateTables
+		\note Enter the name of the System Second Fuel Consumption Lookup Table for Mode0.
+		\note If this field is blank, Mode0 does not consume a second fuel.
+	A25,	\field Mode0 System Third Fuel Consumption Lookup Table Name
+		\type object-list
+		\object-list MultiVariateTables
+		\note Enter the name of the System Third Fuel Consumption Lookup Table for Mode0.
+		\note If this field is blank, Mode0 does not consume a third fuel.
+	A26,	\field Mode0 System Water Use Lookup Table Name
+		\type object-list
+		\object-list MultiVariateTables
+		\note Enter the name of the System Water Use Lookup Table for Mode0.
+		\note If this field is blank, Mode0 does not consume water.
+	N6,	\field Mode0 Outside Air Fraction
+		\type real
+		\minimum 0.00
+		\maximum 1.00
+		\note Enter the outside air fraction for Mode0.
+		\note If this field is blank, the outside air fraction for Mode0 will be 0.00.
+		\default 0.00
+	N7,	\field Mode0 Supply Air Mass Flow Rate Ratio
+		\type real
+		\minimum 0.00
+		\maximum 1.00
+		\note Enter the supply air mass flow rate ratio for Mode0. The value in this field will be used to determine the supply air mass flow rate in Mode0.
+		\note Supply air mass flow rate ratio describes supply air mass flow rate as a fraction of mass flow rate associated with the value in field: "System Maximum Supply Air Flow Rate".
+		\note If this field is blank, the supply air mass flow rate ratio for Mode0 will be 0.
+		\default 0.00
+	A27,	\field Mode1 Name
+		\type alpha
+		\retaincase
```

Comment:
The mode name is to be available as a user interpreted output. For example it is clearer to write that the mode in each time step was "IEC and DX1" than it is to write "iec and dx1".   I may be mistaking the purpose of retaincase.  Is there a reason to not retain case?